pkg/lib/authn/identity/ldap_identity.go

```diff
@@ -18,6 +18,14 @@ type LDAP struct {
 	RawEntryJSON         map[string]interface{} `json:"raw_entry_json,omitempty"`
 }
 
+func (i *LDAP) DisplayID() string {
+	dn, ok := i.RawEntryJSON["dn"].(string)
+	if !ok {
+		return i.UserIDAttributeValue
```

Comment:
Write this

```
(&ldap.AttributeTypeAndValue{
  Type: i.UserIDAttributeName,
  Value: i.UserIDAttributeValue,
}).String()
```

We want to show `uid=foobar` instead of `foobar`. The former is more informative.